.github/workflows/test.yaml

```diff
@@ -59,6 +59,8 @@ jobs:
 
       - name: Run Go tests 🔬
         run: go test -p 1 -cover -covermode atomic -coverprofile=profile.cov -v ./...
+        env:
+          GITHUB_AUTH_TOKEN: ${{ secrets.INTEGRATION }}
```

Comment:
This is to increase the rate limit of the GitHub API, so that consecutive CI runs don't hit the limit.